hardhat.config.js

```diff
@@ -27,21 +27,22 @@ const optimizerSettings = {
         },
     },
 };
-const compilerSettings = {
+// For contracts that are fixed to a specific version, we fix the compiler settings as well
+const fixedContractCompilerSettings = {
     version: '0.8.21',
     settings: {
         evmVersion: process.env.EVM_VERSION || 'london',
-        optimizer: optimizerSettings,
+        optimizer: {
+            ...optimizerSettings,
+            runs: 1000,
+        },
     },
 };
-const itsCompilerSettings = {
-    version: '0.8.24',
+const compilerSettings = {
+    version: '0.8.27',
```

Comment:
Why we are bumping up to `0.8.27`? I looked up the ticket [AXE-6240](https://axelarnetwork.atlassian.net/browse/AXE-6240) but couldn't find the exact reason.

Reply:
I reviewed the solidity releases, they look more stable than earlier ones in 0.8.x line. 0.8.27 also adds support returning custom errors via the `require` statement.